pkg/registry/core/node/storage/storage.go

```diff
@@ -72,6 +73,28 @@ func (r *StatusREST) Update(ctx genericapirequest.Context, name string, objInfo
 	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation)
 }
 
+// ConfigSourceREST implements the configsource subresource for a Node
+type ConfigSourceREST struct {
+	Store *genericregistry.Store
+}
+
+var (
+	_ = rest.Getter(&ConfigSourceREST{})
+	_ = rest.Updater(&ConfigSourceREST{})
+)
+
+func (r *ConfigSourceREST) New() runtime.Object {
+	return &api.Node{}
+}
+
+func (r *ConfigSourceREST) Get(ctx genericapirequest.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
```

Comment:
I would go ahead and leave this out unless you have a need for it. It's not accessible from the client as is.

Reply:
sounds good